src/master-playlist-controller.js

```diff
@@ -370,7 +336,7 @@ export default class MasterPlaylistController extends videojs.EventTarget {
     // Pause any alternative audio
     if (this.audioPlaylistLoader_) {
       this.audioPlaylistLoader_.pause();
-      this.audioPlaylistLoader_ = null;
+      this.audioPlaylistLoader_.dispose();
```

Comment:
Isn't audioPlaylistLoader_ a separate reference? Setting it to null shouldn't modify the entry in audioPlaylistLoaders_. It should just stop pointing to the entry.

The benefit to keeping them around and not re-creating them is that we save on time to start if the playlist is already loaded. Not a 100% requirement, but it is helpful.